app/frontend/src/components/designer/settings/FormScheduleSettings.vue

```diff
@@ -577,11 +577,13 @@ defineExpose({
           </b>
         </span>
 
-        <span :lang="locale">{{
+        <span :lang="locale" data-test="late-submission-text">{{
           form.schedule.allowLateSubmissions.enabled &&
           form.schedule.allowLateSubmissions.forNext.intervalType &&
           form.schedule.allowLateSubmissions.forNext.term
-            ? $t('trans.formSettings.allowLateSubmissnInterval') +
+            ? ' ' +
+              $t('trans.formSettings.allowLateSubmissnInterval') +
+              ' ' +
```

Comment:
Hmm, I think this is adding an extra space in this case:
![image](https://github.com/user-attachments/assets/ee63b4a8-1779-4e63-a2a6-fbc0c3f336d5)
Looks like there's also another missing space around line 605 for `1months`?

Reply:
Good catch! Its fixed now. 

![Screenshot 2024-08-19 at 11 35 49 AM](https://github.com/user-attachments/assets/661cb741-1251-454c-81ba-527438bd6b3b)